src/main/java/com/example/greeting/controller/HelloController.java

```diff
@@ -0,0 +1,13 @@
+package com.example.greeting.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+
+@Controller
+public class HelloController {
+    @GetMapping("/hello")
+    private String hello() {
+        return "/index";
```

Comment:
```suggestion
        return "index";
```
ファイル名(拡張子なし)が望ましいです！

Reply:
承知しました！
ありがとうございます。

Reply:
[ef0ffba](https://github.com/yoko-newDeveloper/raiseTech-course-task6/pull/2/commits/ef0ffba5d0604196aea2a18aa9171c1c85892776)
昨日対応してpushしました。

下記確認させてください。
それぞれのキャプチャを添付します。
【確認事項】
・GitHub：Controllerがない
・ローカルディレクトリ：Contorollerが昨夜21:03の変更が保存されている
・IntelliJ：L10の状態で保存されている
・ターミナル：git statusやls -aで確認しても、変更は登録されていると考えられる
【疑問点】
・GitHubにControllerがない？
・修正がレビュア小山さんには確認できない？

![93FD9FE2-8561-4F78-824D-4F07DBBC6682](https://github.com/yoko-newDeveloper/raiseTech-course-task6/assets/91002836/8ae2278c-ff3b-4989-a353-068afd082610)

![05280797-46F2-460C-AFD5-2EE237023C83_1_201_a](https://github.com/yoko-newDeveloper/raiseTech-course-task6/assets/91002836/6985340c-9c71-4905-8b98-dadecc797808)

![0A030144-4593-43B9-AB5A-B8F644A51479_1_201_a](https://github.com/yoko-newDeveloper/raiseTech-course-task6/assets/91002836/054420e1-5c10-4731-b3b9-8527536c2388)

![898A65FD-88E1-46F4-BBC6-7ED6FB066469_1_201_a](https://github.com/yoko-newDeveloper/raiseTech-course-task6/assets/91002836/f6b5ba01-0c75-44f4-b4f0-cfa9d079855b)